sktime/forecasting/base/adapters/_generalised_statsforecast.py

```diff
@@ -504,7 +504,7 @@ def predict(self, h, X=None, level=None):
             self.prediction_intervals = ConformalIntervals(h=h)
 
         level = sorted(level)
-        coverage = [round(1 - (_l / 100), 2) for _l in level]
+        coverage = [round(_l / 100, 2) for _l in level]
```

Comment:
agreed, I did not spot this in the original review.

Although it might cause issues with how `statsforecast` names the columns, in case we use this later in indexing.